tests/std/tests/P0943R6_stdatomic_h/test.compile.pass.cpp

```diff
@@ -0,0 +1,111 @@
+static_assert(is_same_v<std::atomic_uint16_t, atomic_uint16_t>);
+static_assert(is_same_v<std::atomic_int32_t, atomic_int32_t>);
+static_assert(is_same_v<std::atomic_uint32_t, atomic_uint32_t>);
+static_assert(is_same_v<std::atomic_int64_t, atomic_int64_t>);
+static_assert(is_same_v<std::atomic_uint64_t, atomic_uint64_t>);
+static_assert(is_same_v<std::atomic_int_least8_t, atomic_int_least8_t>);
+static_assert(is_same_v<std::atomic_uint_least8_t, atomic_uint_least8_t>);
+static_assert(is_same_v<std::atomic_int_least16_t, atomic_int_least16_t>);
+static_assert(is_same_v<std::atomic_uint_least16_t, atomic_uint_least16_t>);
+static_assert(is_same_v<std::atomic_int_least32_t, atomic_int_least32_t>);
+static_assert(is_same_v<std::atomic_uint_least32_t, atomic_uint_least32_t>);
+static_assert(is_same_v<std::atomic_int_least64_t, atomic_int_least64_t>);
+static_assert(is_same_v<std::atomic_uint_least64_t, atomic_uint_least64_t>);
+static_assert(is_same_v<std::atomic_int_fast8_t, atomic_int_fast8_t>);
+static_assert(is_same_v<std::atomic_uint_fast8_t, atomic_uint_fast8_t>);
+static_assert(is_same_v<std::atomic_int_fast16_t, atomic_int_fast16_t>);
+static_assert(is_same_v<std::atomic_uint_fast16_t, atomic_uint_fast16_t>);
+static_assert(is_same_v<std::atomic_int_fast32_t, atomic_int_fast32_t>);
+static_assert(is_same_v<std::atomic_uint_fast32_t, atomic_uint_fast32_t>);
+static_assert(is_same_v<std::atomic_int_fast64_t, atomic_int_fast64_t>);
+static_assert(is_same_v<std::atomic_uint_fast64_t, atomic_uint_fast64_t>);
+static_assert(is_same_v<std::atomic_intptr_t, atomic_intptr_t>);
+static_assert(is_same_v<std::atomic_uintptr_t, atomic_uintptr_t>);
+static_assert(is_same_v<std::atomic_size_t, atomic_size_t>);
+static_assert(is_same_v<std::atomic_ptrdiff_t, atomic_ptrdiff_t>);
+static_assert(is_same_v<std::atomic_intmax_t, atomic_intmax_t>);
+static_assert(is_same_v<std::atomic_uintmax_t, atomic_uintmax_t>);
+
+namespace test {
+    using ::atomic_compare_exchange_strong;
+    using ::atomic_compare_exchange_strong_explicit;
+    using ::atomic_compare_exchange_weak;
+    using ::atomic_compare_exchange_weak_explicit;
+    using ::atomic_exchange;
+    using ::atomic_exchange_explicit;
+    using ::atomic_fetch_add;
+    using ::atomic_fetch_add_explicit;
+    using ::atomic_fetch_and;
+    using ::atomic_fetch_and_explicit;
+    using ::atomic_fetch_or;
+    using ::atomic_fetch_or_explicit;
+    using ::atomic_fetch_sub;
+    using ::atomic_fetch_sub_explicit;
+    using ::atomic_flag_clear;
+    using ::atomic_flag_clear_explicit;
+    using ::atomic_flag_test_and_set;
+    using ::atomic_flag_test_and_set_explicit;
+    using ::atomic_is_lock_free;
+    using ::atomic_load;
+    using ::atomic_load_explicit;
+    using ::atomic_store;
+    using ::atomic_store_explicit;
+} // namespace test
+
+static_assert(std::atomic_thread_fence == atomic_thread_fence);
+static_assert(std::atomic_signal_fence == atomic_signal_fence);
+
+int main() {}

```

Comment:
In `test.compile.pass.cpp`, we conventionally add a comment as an extra reminder:

https://github.com/microsoft/STL/blob/a9321cfe53ea31a7e197d5d8336167d6ca3de8b6/tests/std/tests/Dev11_0272959_make_signed/test.compile.pass.cpp#L11